src/autocomplete.css

```diff
@@ -29,6 +29,14 @@
   outline: 3px solid #ffbf47;
 }
 
+.autocomplete__input.autocomplete__input--show-all-values {
```

Comment:
I don't think we don't need the extra specificity here of the `.autocomplete__input` selector?